src/actions/content.js

```diff
@@ -0,0 +1,32 @@
+const CONTENT_START_LOADING = 'CONTENT_START_LOADING'
+const CONTENT_FETCHED = 'CONTENT_FETCHED'
+const CONTENT_FETCHED_ERROR = 'CONTENT_FETCHED_ERROR'
+const CONTENT_ADD_FAV = 'CONTENT_ADD_FAV'
+const CONTENT_REMOVE_FAV = 'CONTENT_REMOVE_FAV'
+const CONTENT_SEARCH_START = 'CONTENT_SEARCH_START'
+
+const startLoading = (payload = {}) => ({type: CONTENT_START_LOADING, payload})
```

Comment:
👍 , you can also create your own `createAction` helper, or use the one provided by `redux-action`.